styles/base.css

```diff
@@ -464,86 +464,118 @@
   /* ** TYPOGRAPHY ** */
 
   /* fonts */
-  --body-font-family: roboto, roboto-fallback, sans-serif;
-  --heading-font-family: roboto-condensed, roboto-condensed-fallback, sans-serif;
-  --spectrum-sans-serif-font-family: Adobe Clean;
-  --spectrum-serif-font-family: Adobe Clean Serif;
+  --spectrum-display-font-family: Adobe Clean Display;
+  --spectrum-display-font-family-stack: adobe-clean-display, var(--spectrum-display-font-family), "Source Sans Pro", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Ubuntu, "Trebuchet MS", "Lucida Grande", sans-serif;
+  --spectrum-display-font: var(--spectrum-display-font-family-stack);
 
+  --spectrum-sans-serif-font-family: Adobe Clean;
   --spectrum-sans-font-family-stack: adobe-clean,
-    var(--spectrum-sans-serif-font-family), "Source Sans Pro", -apple-system,
-    BlinkMacSystemFont, "Segoe UI", Roboto, Ubuntu, "Trebuchet MS",
-    "Lucida Grande", sans-serif;
+  var(--spectrum-sans-serif-font-family), "Source Sans Pro", -apple-system,
+  BlinkMacSystemFont, "Segoe UI", Roboto, Ubuntu, "Trebuchet MS",
+  "Lucida Grande", sans-serif;
   --spectrum-sans-serif-font: var(--spectrum-sans-font-family-stack);
-
+  
+  --spectrum-serif-font-family: Adobe Clean Serif;
```

Comment:
I wish these variables were set up the other way, like 

```css
--font-fallback-serif: "Source Sans Pro", -apple-system,
  BlinkMacSystemFont, "Segoe UI", Roboto, Ubuntu, "Trebuchet MS",
  "Lucida Grande", sans-serif;
  
--font-primary: 'Adobe Clean', var(--font-fallback-serif); 

```

Reply:
but we should follow spectrum

Reply:
Yeah, I agree but that's the conclusion I landed at as well.